mlrun/config.py

```diff
@@ -405,17 +409,45 @@ def get_hub_url():
         return config.hub_url
 
     @staticmethod
-    def get_default_function_node_selector():
-        default_function_node_selector = {}
-        if config.default_function_node_selector:
-            default_function_node_selector_json_string = base64.b64decode(
-                config.default_function_node_selector
-            ).decode()
-            default_function_node_selector = json.loads(
-                default_function_node_selector_json_string
-            )
+    def decode_base64_config_and_load_to_dict(attribute_name: str):
+        parts = (
+            attribute_name.split(".")
+            if isinstance(attribute_name, str)
+            else attribute_name
```

Comment:
What's the scenario of attribute name not being a string you're trying to handle it here ?
Sounds to me like we should just explode in that case

Reply:
no scenerio, removed.